doc/source/example-newsreader.rst

```diff
@@ -0,0 +1,27 @@
+News Reader
+===========
+
+This document shows how to implement a simple news reader using Ray. The reader
+consists of a simple Vue.js `frontend`_ and a backend consisting of a Flask
+server and a Ray actor. View the `code for this example`_.
+
+To run this example, you will need to install NPM and a few python dependencies.
+
+.. code-block:: bash
+
+  pip install atoma
+  pip install flask
+
+
+To use this example you need to
+
+* Start the server with ``python server.py``
+* Clone the client code with ``git clone https://github.com/ray-project/qreader``
+* Start the client with ``cd qreader;  npm install; npm run dev``
+* You can now add a channel by clicking "Add channel" and for example pasting
+  ``http://news.ycombinator.com/rss`` into the field.
+* Star some of the articles and dump the database with ``sqlite3 newsreader.db``
```

Comment:
Make it clear that this should be run from a terminal. I tried pasting this into the search bar in the web app.

Reply:
done